yarn-project/simulator/src/avm/journal/journal.ts

```diff
@@ -67,11 +93,28 @@ export class AvmPersistableStateManager {
   public newL1Messages: L2ToL1Message[] = [];
   public newLogs: UnencryptedL2Log[] = [];
 
+  // TRANSITIONAL: This should be removed once the AVM is fully operational and the public executor is gone.
+  public transitionalExecutionResult: PartialPublicExecutionResult;
+
   constructor(hostStorage: HostStorage, parent?: AvmPersistableStateManager) {
     this.hostStorage = hostStorage;
     this.publicStorage = new PublicStorage(hostStorage.publicStateDb, parent?.publicStorage);
     this.nullifiers = new Nullifiers(hostStorage.commitmentsDb, parent?.nullifiers);
     this.trace = new WorldStateAccessTrace(parent?.trace);
+
+    this.transitionalExecutionResult = {
+      nullifierReadRequests: [],
+      nullifierNonExistentReadRequests: [],
+      newNoteHashes: [],
+      newL2ToL1Messages: [],
+      startSideEffectCounter: this.trace.accessCounter,
+      newNullifiers: [],
+      contractStorageReads: [],
+      contractStorageUpdateRequests: [],
+      unencryptedLogs: [],
```

Comment:
Miranda recently added some new log structures so I think this needs updating: https://app.graphite.dev/github/pr/AztecProtocol/aztec-packages/5934/feat-hash-logs-inside-circuit?panel=files#file-yarn-project/simulator/src/public/executor.ts

Reply:
:+1: good catch, addressed some of this and created a couple of follow-up tickets #6205 and #6206